designing-reports/report-designer-tools/web-report-designer/tools/wrd-htmltextbox-editor.md

```diff
@@ -0,0 +1,35 @@
+---
+title: HtmlTextBox Editor
+page_title: Editing the HtmlTextBox Report Item
+description: "Learn how to edit the content of the Telerik Reporting HtmlTextBox report item in the Web Report Designer."
+slug: wrd-htmltextbox-editor
+tags: telerik, reporting, report, items, htmltextbox, editing, editor, web report designer
+published: True
+position: 9
+---
+
+# Editing in HtmlTextBox
+
+In [2024 Q1 (18.0.24.130)](https://www.telerik.com/support/whats-new/reporting/release-history/progress-telerik-reporting-2024-q1-18-0-24-130) we replaced the plain-text editor for the `Value` property of the [HtmlTextBox item]({%slug telerikreporting/designing-reports/report-structure/htmltextbox/overview%}) with a [KendoUI Editor component](https://docs.telerik.com/kendo-ui/controls/editor/get-started). This lets the user have instant visual feedback on how their HTML will look in the rendered report.
+
+The editor lets you easily add [Embedded Expressions in HtmlTextBox]({%slug htmltextbox_embedded_expressions%}) through the list of predefined and dynamic elements that can be used in your expression. A list of expression element categories is located in the lower left-hand list of the editor, the elements for each category are located in the lower middle list, and descriptions are displayed in the lower right-hand panel.
+
+![Look at the HtmlTextBox editor of the Web Report Designer.](images/wrd-htmltextbox-editor-screenshot.png)
+
+To add an embedded expression to the HtmlTextBox content double-click the list of elements in the bottom middle list. This will add the corresponding dynamic object wrapped in curly brackets. You may further edit the content inside the brackets manually by adding other dynamic objects and relations between them. The entire embedded expression will be evaluated and replaced in runtime by the Reporting engine.
```

Comment:
Isn't "double-click an element from the bottom middle list" more accurate than "double-click the list of elements in the bottom middle list" here?